public/stylesheets/style.css

```diff
@@ -0,0 +1,65 @@
+body {
+  font: 14px "Lucida Grande", Helvetica, Arial, sans-serif;
+  margin: 0;
+  min-width: 960px;
+  border-top: 10px solid #b71f39;
+
+}
+
+.grapes {
+  width: 100%;
+  height: 100%;
+}
+
+.searchWrapper {
+  margin-left: 5%;
+  display: inline;
+  justify-content: center;
+}
+
+.searchBox{
+  -moz-border-radius: 5px;
+  -webkit-border-radius: 5px;
+  border-radius: 5px;
+  border: 1px solid lightgrey;
+  width: 300px;
+  height: 20px;
+}
+
+.logo{
+  padding: 30px;
+  display: block;
+  margin: auto;
+}
+
+
+.logoContainer{
+  border-bottom: 1px solid #e6e6e6;
+}
+
+.menuItem{
```

Comment:
What font do you plan on using? Also, is this initial setup just meant as a template, and could you update your readme as to the goal of this set up?